text-zoom/ios/Plugin/TextZoom.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+@objc public class TextZoom: NSObject {
+    static var baseFontSize: Double = 17
+
+    @objc public func getPreferred() -> Double {
```

Comment:
`preferredFontSize` or something similar would be a more idiomatic name for internal (i.e. not exposed to JS) methods like this.